package/magento-gcp/xrd.yaml

```diff
@@ -20,6 +20,9 @@ spec:
                 projectId:
                   type: string
                   description: ID of the Google Cloud Platform project.
+                ingressIP:
+                  type: string
+                  description: Ingress IP.
```

Comment:
@ca7alindev Strange changes, what's idea to create GCP address, by providing IP address manually, where i will take it, when i will create this XR?